nuxeo-distribution/nuxeo-functional-tests/src/main/java/org/nuxeo/functionaltests/pages/tabs/ArchivedVersionsSubPage.java

```diff
@@ -45,6 +47,8 @@
  */
 public class ArchivedVersionsSubPage extends DocumentBasePage {
 
+    private static final Log log = LogFactory.getLog(ArchivedVersionsSubPage.class);
```

Comment:
Shouldn't we now use Log4j 2?

Reply:
The previous Apache Commons API is still valid. There's no point in stopping using it for new classes (if not needed for performance or clarity reasons) when we still have thousands of existing classes that still use it.